integration-tests/build.gradle.kts

```diff
@@ -37,4 +37,5 @@ subprojects {
 dependencies {
     implementation(kotlin("stdlib"))
     implementation(kotlin("test-junit"))
-}
+    val coroutines_version: String by project
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version")}
```

Comment:
Why do we need `api` here? Wouldn't `implementation` be sufficient?

Reply:
Yes, you are right. I for some reason thought that I am exposing some coroutines stuff in the API, but nope!
Changed to `implementation` ✅